server/authorization/service.go

```diff
@@ -182,12 +184,15 @@ func (s *Service) GetTokenID(tokenFilter string) string {
 func (s *Service) updateTokens(payload []byte) {
 	tokens, err := parseFromBytes(payload)
 	if err != nil {
+		metrics.ErrorTokenUpdating()
```

Comment:
Better rename both to ErrorAPIKeyUpdating SuccessAPIKeyUpdating.
That is how we used to call this entity

Reply:
Renamed